data_override/data_override.F90

```diff
@@ -115,6 +115,29 @@ module data_override_mod
    integer            :: region_type
 end type data_type
 
+type nudge3D
+   character(len=3)                 :: gridname
+   character(len=128)               :: varname
+   character(len=128)               :: obs_coefficient_name
+   character(len=128)               :: mod_coefficient_name
+   real, _ALLOCATABLE               :: model_input(:,:,:) _NULL
+   real, _ALLOCATABLE               :: nudge_output(:,:,:) _NULL
+   real, _ALLOCATABLE               :: observed_data(:,:,:) _NULL
+   real, _ALLOCATABLE               :: observation_coefficient(:,:,:) _NULL
+   real, _ALLOCATABLE               :: model_coefficient(:,:,:) _NULL
+end type nudge3D
+
+type nudge2D
+   character(len=3)                 :: gridname
+   character(len=128)               :: varname
+   character(len=128)               :: obs_coefficient_name
+   character(len=128)               :: mod_coefficient_name
+   real, _ALLOCATABLE               :: model_input(:,:) _NULL
+   real, _ALLOCATABLE               :: nudge_output(:,:) _NULL
+   real, _ALLOCATABLE               :: observed_data(:,:) _NULL
+   real, _ALLOCATABLE               :: observation_coefficient(:,:) _NULL
+   real, _ALLOCATABLE               :: model_coefficient(:,:) _NULL
```

Comment:
This is the same comment I have above about the `_ALLOCATABLE` and `_NULL` statements.  They should be consistent at least.

Reply:
yerp